apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonVariantTestSection.tsx

```diff
@@ -42,6 +42,10 @@ export const ButtonVariantTest: React.FunctionComponent = () => {
       <CompoundButton appearance="subtle" secondaryContent="Compound" style={commonTestStyles.vmargin}>
         Subtle
       </CompoundButton>
+      <CompoundButton secondaryContent="secondaryContent" size="large" style={commonTestStyles.vmargin}>
+        Compound Button
+        <Text>Child element</Text>
```

Comment:
We probably won't officially support this scenario, so I would be okay with not having this test case.